queue/python/queue.py

```diff
@@ -0,0 +1,40 @@
+# Implementation of queue using Linked List:
+class Node:
+    def __init__(self, data):
+       self.data = data
+       self.next = None
+class Queue:
+    def __init__(self):
+        self.head = None
+        self.last = None
+    def enqueue(self, data):
+        if self.last is None:
+            self.head = Node(data)
+            self.last = self.head
+        else:
+            self.last.next = Node(data)
+            self.last = self.last.next
+    def dequeue(self):
+        if self.head is None:
+            return None
+        else:
+            to_return = self.head.data
+            self.head = self.head.next
+            return to_return
+ a_queue = Queue()
+while True:
+    print('enqueue <value>')
+    print('dequeue')
+    print('quit')
+    do = input('What would you like to do? ').split()
+     operation = do[0].strip().lower()
```

Comment:
E901 IndentationError: unindent does not match any outer indentation level

*Origin: PycodestyleBear (E901), Section: `all.autopep8`.*